docs/provider/hashicorp-vault.md

```diff
@@ -43,6 +43,24 @@ Then create a simple k/v pair at path `secret/foo`:
 vault kv put secret/foo my-value=s3cr3t
 ```
 
+If kv engine is not enabled then enable using:
+
+```
+vault secrets enable -path=secret kv-v2
+```
+
+If the KV version is version:1, upgrade it to version:2:
+
+```
+vault kv enable-versioning secret/
+```
```

Comment:
This is a little bit misguidance as we do support KV version to be `v1` - I would rather remove this block just to make sure people use it as an example.

Reply:
Okay, if that is the case, how about if add version: "v1" by default in SecretStore manifest file, that would be easy to follow along.

Reply:
The thing is that vault for development use by default creates a v2 `secret` kv engine, so having that as a default seems more useful at least to me. Maybe adding that in the example might be more meaningful here